system/hostsfile/main.go

```diff
@@ -26,38 +17,6 @@ func confirmRemove(hostname string) bool {
 	return c
 }
 
-// AddEntry - Add an entry to /etc/hosts or equivalent
-func AddEntry(hostname string) error {
-	hosts, err := NewHosts()
-	if err != nil {
-		return err
-	}
-
-	if !hosts.Has(localhost, hostname) {
-		if confirmAmend(hostname) == false {
-			return nil
-		}
-
-		hosts.Add(localhost, hostname)
-		if hosts.IsWritable() == false {
-			err := hosts.WriteElevated()
-			if err != nil {
-				return err
-			}
-
-			return nil
-		}
-
-		if err := hosts.Flush(); err != nil {
-			return err
-		}
-
-		return nil
-	}
-
-	return nil
-}
-
```

Comment:
Really glad we're able to chop out the hostsfile stuff, will make it much easier on cross platform compat